packages/sdk/contractkit/src/base.ts

```diff
@@ -13,10 +10,8 @@ export enum CeloContract {
   EpochManagerEnabler = 'EpochManagerEnabler',
   FederatedAttestations = 'FederatedAttestations',
   FeeCurrencyDirectory = 'FeeCurrencyDirectory',
-  FeeCurrencyWhitelist = 'FeeCurrencyWhitelist',
   FeeHandler = 'FeeHandler',
   Freezer = 'Freezer',
-  GasPriceMinimum = 'GasPriceMinimum',
   /* @deprecated use CeloToken */
   GoldToken = 'GoldToken',
```

Comment:
this has been deprecated forever, what about truly getting rid of it since we're already major bumping... ?

Reply:
it hasnt actually been that long since it was only in march that CeloToken string was added to the registery. 

still we could remove and push people towards using the new CeloToken identifier

Reply:
fair